synthtool/gcp/common.py

```diff
@@ -90,6 +90,9 @@ def py_samples(self, **kwargs) -> List[Path]:
             ]
         )
 
+        # ensure samples will generate
+        kwargs["metadata"]["samples"] = True
```

Comment:
what happens if this is not set?

Reply:
if it isn't set, _generic_library() will add README.md to the `excludes` https://github.com/googleapis/synthtool/blob/ca275fc17e8e92c81c04f19de1d22e016be3f0e2/synthtool/gcp/common.py#L57